content/docs/06-integrations/00-kubernetes.md

```diff
@@ -16,118 +16,567 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 # Kubernetes
 
-This pack defines the properties that will be used by Spectro Cloud to bring up the Kubernetes cluster. Most of the Kubernetes hardening standards recommended by the Center for Internet Security is enabled by default.
+The Kubernetes pack defines the default properties that we use to deploy Kubernetes clusters. Most of the Kubernetes hardening standards that the Center for Internet Security (CIS) recommends are enabled by default. The Kubernetes pack supports most [cloud and data center infrastructure providers](/clusters).
 
-<WarningBox>
-Info: Only Kubernetes versions 1.19 and above is supported
-</WarningBox>
+Spectro Cloud offers Cloud Native Computing Foundation (CNCF) Kubernetes as a core infrastructure pack in Palette. We release Kubernetes updates as follows:
+
+<br />
+
+- Major versions are assessed based on the extent of changes.
+- Minor versions are provided within eight weeks of a new Kubernetes release.
+- Patch versions are provided within four weeks of a new Kubernetes release.
 
-## Supported Kubernetes versions
+In the period of time following a Kubernetes release, the Kubernetes pack undergoes rigorous vulnerability scans and penetration testing. 
 
-<InfoBox>
+A minor Kubernetes version will be deprecated in Palette when the Kubernetes community announces the version is entering End of Life (EOL). The deprecated version will remain in that state for three months and will be disabled after a quarter and deleted from Palette in the following quarter.
```

Comment:
Should we explain somewhere what is meant by deprecate , disable and delete? This is not yet available on the UI as of now and in proposal stage.

Reply:
@vishwanaths This information is now added and will be moved to a separate doc about deprecated packs.